userspace/libsinsp/sinsp_ppm_sc.cpp

```diff
@@ -137,7 +137,6 @@ std::unordered_set<uint32_t> sinsp::enforce_simple_ppm_sc_set(std::unordered_set
 		PPM_SC_TGKILL,
 		PPM_SC_TIMERFD_CREATE,
 		PPM_SC_TKILL,
-		PPM_SC_UMOUNT,
```

Comment:
`PPME_SYSCALL_UMOUNT_E` and `PPME_SYSCALL_UMOUNT_X` modify the state but these events are not used by the `umount` syscall which sends to userspace generic events! So no reason to keep the `umount` syscall in this set. This fact causes the `enforce_sinsp_state_basic` test to fail because when scap tries to recover the necessary sinsp state, it doesn't retrieve the `umount` syscall while we searched it in our tests as you can see here https://github.com/falcosecurity/libs/pull/620/commits/b7023d6b52d2253f81900f680fabc97e284b0e0e#diff-392077484120e947331167650083cf984c41375069a32c252951657400a716a6L244-L247